docs/extensions/datacollector-migration.md

```diff
@@ -0,0 +1,44 @@
+# Migrate your TestPlatform v0(TPv0)/TestPlatoformv1(TPv1) datacollector to TestPlatform v2(current version) datacollector
+DataCollectors were supported in TPv0(mstest.exe)/TPv1(vstest.console.exe). 
+This document will walk you through the changes that are required to migrate your TPv0/TPv1 DataCollector to work with TPv2.
+
+## Referencing DataCollector Framework
```

Comment:
Just start by saying "The Visual Studio Test Platform has introduced differences and enhancements to the data collection infrastructure. If you already have written data collectors, then you will need to migrate them as shown in this document."